70_FLIPFluids/app_resources/compute/advectParticles.comp.hlsl

```diff
@@ -0,0 +1,36 @@
+#include "../common.hlsl"
+#include "../gridSampling.hlsl"
+#include "../descriptor_bindings.hlsl"
+
+[[vk::binding(b_apGridData, s_ap)]]
+cbuffer GridData
+{
+    SGridData gridData;
+};
```

Comment:
We tend to use BDA for everything but this is actually a use of an UBO I can commend, globals that don't change, ever

(although it would probably fit in push constants - which are slightly faster)